europe/README.EUROPE

```diff
@@ -457,4 +457,26 @@ only used as steps in more complicated transformations.
 
 Init file with various transformations and conversions of relevance in Iceland.
 
-* ISL
\ No newline at end of file
+* ISL
+
+### Hungary: ETRS89 HD72/EOV (EPSG:23700)
+
+*Source*: [Department of Geodesy and Surveying of Budapest University of Technology and Economics](https://github.com/OSGeoLabBp/eov2etrs)  
+*Format*: NTv2
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Grid transformation from ETRS89 to HD72/EOV.
+
+* etrs2eov_notowgs.gsb
+
+### Hungary: ETRS89 heights -> Baltic heights
```

Comment:
which of the baltic heights ? EPSG references 3 versions: EPSG:5705 (Baltic 1977 height) or EPSG:5786 (Baltic 1982 height) or 'EPSG:8357 (Baltic 1957 height)